platform/ui/src/components/IconButton/IconButton.jsx

```diff
@@ -87,15 +87,16 @@ const IconButton = ({
   disabled = defaults.disabled,
   type = defaults.type,
   fullWidth = defaults.fullWidth,
+  onClick,
   className,
   ...rest
 }) => {
   const buttonElement = useRef(null);
 
   const handleOnClick = (e) => {
     buttonElement.current.blur();
-    if (rest.onClick) {
-      rest.onClick(e);
+    if (onClick) {
+      onClick(e);
```

Comment:
I sometimes see a default `no operation` function for things like this, so we don't have to litter in checks:

```onClick: () => {}```

Reply:
I've fixed the `defaultProps` definition for this component and created a default method for `onClick`. Thanks!